src/MockClassifier.Api/Controllers/InstitutionController.cs

```diff
@@ -1,12 +1,25 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using MockClassifier.Api.Interfaces;
 using MockClassifier.Api.Models;
+using MockClassifier.Api.Services.Dmr;
 
 namespace MockClassifier.Api.Controllers
 {
     [Route("input-from-dmr/institution")]
     [ApiController]
     public class InstitutionController : ControllerBase
     {
+        private readonly IDmrService _dmrService;
+        private readonly ITokenService _tokenService;
+        private readonly INaturalLanguageService _naturalLanguageService;
```

Comment:
Since I'm new to C#/.NET, just checking - does the I in front of like IDmrService mean "Interface"?

Reply:
I see, just a sanity check for me then